src/Core/Entity/Webhook.php

```diff
@@ -0,0 +1,60 @@
+<?php
+
+/**
+ * Ushahidi Webhook Entity
+ *
+ * @author     Ushahidi Team <team@ushahidi.com>
+ * @package    Ushahidi\Platform
+ * @copyright  2014 Ushahidi
+ * @license    https://www.gnu.org/licenses/agpl-3.0.html GNU Affero General Public License Version 3 (AGPL3)
+ */
+
+namespace Ushahidi\Core\Entity;
+
+use Ushahidi\Core\StaticEntity;
+
+class Webhook extends StaticEntity
+{
+	protected $id;
+	protected $user_id;
+	protected $post_id;
```

Comment:
Why does a webhook have a post id? Shouldn't a webhook just be a URL, secrets and the user id that registered it (maybe)?